src/psyclone/psyir/tools/read_write_info.py

```diff
@@ -120,7 +123,9 @@ def all_used_vars_list(self):
         return all_vars
 
     # -------------------------------------------------------------------------
-    def add_read(self, signature, container_name=None):
+    def add_read(self,
+                 signature: Signature,
+                 container_name: Optional[str] = None) -> None:
         '''This function adds a read access to the specified signature and
         container name. The container_name is optional and defaults to "",
         indicating that this signature is not based on importing a symbol
```

Comment:
Please rm the `:type ...` parts of this docstring.

Reply:
Done.